example/src/main/java/com/github/appintro/example/ui/custom/MarginsIntro.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.appintro.example.ui.custom
+
+import android.os.Bundle
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.github.appintro.AppIntro
+import com.github.appintro.AppIntroFragment
+import com.github.appintro.example.R
+import com.github.appintro.example.ui.custom.fragments.MarginsSlideFragment
+
+class MarginsIntro : AppIntro() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // Set a color to the bar so that it's not transparent
+        setBarColor(ContextCompat.getColor(this, R.color.appintro_example_orange))
+
+        addSlide(
+            AppIntroFragment.createInstance(
+                "Welcome",
+                "This is a demo of the AppIntro library, with setBarMargin set to true."
+            )
+        )
+
+        addSlide(
+            MarginsSlideFragment.newInstance(R.string.margins_hide)
+        )
+
+        addSlide(
+            MarginsSlideFragment.newInstance(R.string.margins_no_hide)
+        )
+    }
+
+    override fun onPageSelected(position: Int) {
+        super.onPageSelected(position)
+
+        // Enable the margin only on the last page
+        setBarMargin(position == 2)
```

Comment:
instead of doing this, could you add a `<CheckBox>` inside the `intro_margins.xml` that toggles the `setBarMargin`?